src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -181,4 +205,46 @@ export default class ReplicaSetDetailController {
    * @private
    */
   onReplicaSetDeleteError_(err) { this.log_.error(err); }
+
+  /**
+   * Applies sorting on pods table.
+   * @param {string} sortBy
+   * @export
+   */
+  sortPods(sortBy) {
```

Comment:
Is it possible to write reasonable tests for sorting if we are using `orderBy` in `ng-repeat` and there is no sort logic inside controller (it seems `orderBy` updates only view and `ctrl.events` is not updated)? Integration tests can cover it, but frontend tests can be difficult to write. Please let me know if you have any idea how to do it.